packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -10,6 +10,15 @@ import '../../../shared/analytics/constants.dart' as gac;
 import '../../../shared/editor/api_classes.dart';
 import '../../../shared/editor/editor_client.dart';
 
+typedef EditableWidgetData =
+    ({List<EditableArgument> args, String? name, String? documentation});
+
+typedef EditArgumentFn =
```

Comment:
Nit: prefer "function" instead of "fn" to avoid abbreviations

Reply:
Done!